src/scratchtocatrobat/converter/converter.py

```diff
@@ -83,20 +82,15 @@ def to_placeholder_brick(self, held_by_block_name=None):
         return [_placeholder_for_unmapped_blocks_to(*self.block_and_args)] if held_by_block_name is None \
                else [_placeholder_for_unmapped_formula_blocks_to(held_by_block_name, *self.block_and_args)]
 
-def _escape_arguments(arguments):
-    for k, v in XML_CHARACTERS_TO_BE_REPLACED_MAPPING.iteritems():
-        arguments = map(lambda arg: arg.replace(k, v) if isinstance(arg, basestring) else arg, arguments)
-    return arguments
-
```

Comment:
I think the `_escape_arguments` function is still necessary since it is intended to escape special XML characters such as <, >, &, etc.
These characters may occur in exception messages of unconvertable bricks which are replaced with a NoteBrick. The exception message is then set as text argument of the NoteBrick in order to show it to the user.